package.json

```diff
@@ -9,11 +9,12 @@
   "scripts": {
     "apidocs": "./node_modules/.bin/jsdoc2md",
     "browser-tests": "NODE_ENV=test ./node_modules/.bin/karma start --browsers Chrome",
-    "build": "NODE_ENV=production npm test && npm run clean && ./node_modules/.bin/webpack --config package.config.js --optimize-minimize",
+    "build": "NODE_ENV=production npm run clean && ./node_modules/.bin/webpack --config package.config.js --optimize-minimize",
     "build-docs": "./update-gh-pages.sh",
     "clean": "./node_modules/.bin/rimraf ./dist/*",
     "lint": "./node_modules/.bin/eslint .storybook/ src/ stories/ test/ *.js",
-    "prepublish": "npm run build",
+    "prepare": "npm run build",
```

Comment:
Ok, there is one weird thing about this config. I chose to run the build only on `prepare` because when you `npm link` it _is_ nice to have certainty that a build will exist in dist/ (e.g., if you forget to `npm start`, which I have done). But `prepare` always runs before `prepublishOnly`, so that means we build, then delete everything, reinstall, and build again. I'm open to removing `prepare` altogether...what do you think @krystophv?